homeassistant/components/habitica/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+    "config": {
+        "error": {
+            "connection_error": "Connection error, seems like Habitica API server is down",
+            "invalid_credentials": "Invalid credentials. Make sure API user ID and API key are correct",
+            "unknown": "Unknown error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "url": "URL to your Habitica instance, if you are hosting your own",
```

Comment:
Please use as many common strings as possible via reference. Here are the common strings:
https://github.com/home-assistant/core/blob/111c2006c8cf37a0d26995c0048c26780f731017/homeassistant/strings.json#L19-L58

Here's how to reference:
https://github.com/home-assistant/core/blob/111c2006c8cf37a0d26995c0048c26780f731017/homeassistant/components/adguard/strings.json#L7